### PR TITLE
test: fix broken auth_mongo test

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

File: .ci/docker-compose-file/conf.env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 EMQX_AUTH__LDAP__SERVERS=ldap_server
-EMQX_AUTH__MONGO__SERVER=mongo_server:27017
```
Comment (zmstone) [inline .ci/docker-compose-file/conf.env:2]: feels like an issue to cause the cases either pass or fail,
why flaky ?

Comment (thalesmg) [reply .ci/docker-compose-file/conf.env:2]: "Flaky" was perhaps the wrong word.  I was getting this failure fairly often (perhaps all the times) :thinking:

Comment (thalesmg) [reply .ci/docker-compose-file/conf.env:2]: Actually, it seems flaky indeed: https://github.com/emqx/emqx/runs/8144558292?check_suite_focus=true

I think that when a self-hosted runner is running our ee5.0 branch containers, they are dirtying the environment:

>  Found orphan containers (redis-sentinel-tls, mongosharded1, redis-tls, mongo3, mongo-tls, mongosharded3, mongo2, mongo1, pgsql-tls, mongo_rs_client, mongosharded_client, mongosharded2, mysql-tls, erlang24, redis-sentinel) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.

 :thinking:

**Comment by thalesmg** [comment]
Apparently, this didn't solve the issue; there are still flaky `nxdomain` errors in CI.

**Pull request closed without merging**